test/AST-Quake/qalloc_initialization.cpp

```diff
@@ -117,17 +117,17 @@ struct RockyRoad {
 // CHECK-DAG:       %[[VAL_8:.*]] = arith.constant 0.000000e+00 : f64
 // CHECK-DAG:       %[[VAL_9:.*]] = cc.alloca f64
 // CHECK:           cc.store %[[VAL_8]], %[[VAL_9]] : !cc.ptr<f64>
-// CHECK:           %[[VAL_10:.*]] = call @_ZNSt8literals16complex_literalsli1iEe(%[[VAL_7]]) : (f{{[1280]+}}) -> complex<f64>
+// CHECK:           %[[VAL_10:.*]] = call @_ZNSt[[NAME1:.*]]8literals16complex_literalsli1i[[NAME2:.*]]Ee(%[[VAL_7]]) : (f{{[1280]+}}) -> complex<f64>
 // CHECK:           %[[VAL_11:.*]] = cc.alloca complex<f64>
 // CHECK:           cc.store %[[VAL_10]], %[[VAL_11]] : !cc.ptr<complex<f64>>
-// CHECK:           %[[VAL_12:.*]] = call @_ZStplIdESt7complexIT_ERKS1_RKS2_(%[[VAL_9]], %[[VAL_11]]) : (!cc.ptr<f64>, !cc.ptr<complex<f64>>) -> complex<f64>
+// CHECK:           %[[VAL_12:.*]] = call @_Z[[NAME3:.*]]7complexIT_[[NAME4:.*]]_(%[[VAL_9]], %[[VAL_11]]) : (!cc.ptr<f64>, !cc.ptr<complex<f64>>) -> complex<f64>
```

Comment:
```suggestion
// CHECK:           %[[VAL_12:.*]] = call @_Z{{.*}}7complexIT_{{:.*}}_(%[[VAL_9]], %[[VAL_11]]) : (!cc.ptr<f64>, !cc.ptr<complex<f64>>) -> complex<f64>
```
This is shorter and somewhat simpler and can be used throughout on the rest of this test.